auth_token/utils.py

```diff
@@ -45,10 +51,14 @@ def login(request, user, expiration=True, auth_slug=None, related_objs=None, bac
     token = Token.objects.create(user=user, user_agent=request.META.get('HTTP_USER_AGENT', '')[:256],
                                  expiration=expiration, auth_slug=auth_slug, ip=get_ip(request),
                                  backend=backend, allowed_cookie=allowed_cookie, allowed_header=allowed_header)
+    if settings.TWO_FACTOR_ENABLED:
+        token.is_two_factor_authenticated = False
+        token.two_factor_key = generate_two_factor_key(Token.TWO_FACTOR_KEY_LENGTH)
```

Comment:
use function make_password to hash token -> password should be hashed. Maybe field should be empty and before calling method for sending code it should be generated sent to the function in raw format and then hashed and stored to the database.